backend/infrastructures/apps/extract/views/serializers.py

```diff
@@ -0,0 +1,5 @@
+from rest_framework import serializers
+
+
+class ExtractDataSerializer(serializers.Serializer):
+    file_path = serializers.CharField(min_length=1)
```

Comment:
File path or file_name?

According to me, it's dangerous to allow user to specify path.
Please, read about `MEDIA` e.g. in Django documentation.

We need to have en_to_end secure test that user can't specify ANY path.